test/blackbox/anonymous_policy.bats

```diff
@@ -1,8 +1,17 @@
-load helpers_pushpull
+load helpers_zot
+
+function verify_prerequisites {
+    if [ ! -f ${ZOT_PATH} ]; then
+        echo "you need to build ${ZOT_PATH} before running the tests" >&3
+        return 1
+    fi
+
+    return 0
```

Comment:
Don't we need to add a check for skopeo here, and in the other files using the skopeo command?